app/models/mediaflux/asset.rb

```diff
@@ -12,6 +12,17 @@ def initialize(id:, name:, collection:, path: nil, last_modified_mf: nil, size:
       @last_modified_mf = last_modified_mf
     end
 
+    def name
```

Comment:
This change was technically not part of the original issue, but our TestGenerator uses Mediaflux's unnamed files so I had to update the code to support them. Which in the end is a good thing since now our code handles these files in case our users were to use this feature in Mediaflux.